sourcemash/frontend/templates/help.tpl

```diff
@@ -0,0 +1,91 @@
+<div class="help-fixed-action-btn">
+  <span class="btn-floating btn-large">?</span>
+</div>
+
+<div id="help-modal" class="modal">
+  <div class="modal-content section no-pad-bot">
+    <div class="container">
+      <div class="row">
+
+        <h2 class="col s12 light center header flow-text">Confused?</h2>
+        <h4 class="col s12 light center header flow-text">Let's get you back on track!</h4>
+        <p class="col s12 divider"><p>
+
+        <h5 class="col s12 light center header flow-text">Sourcemash categorizes your articles to help you get through your feeds faster.</h5>
+
+      </div>
+
+      <div class="row">
+        <div class="col s4">
+          <div class="center promo">
+            <a href="/#browse">
+              <i class="mdi-editor-insert-comment promo-icon"></i>
+            </a>
+          </div>
+        </div>
+        <div class="col s8">
+          <p class="promo-caption">Step 1: Source selection</p>
+          <p class="light">Choose the sources you want to read. Select from our list of feeds on the <a href="/#browse">Browse page</a>, or add your own using the form at the top of the page! Once you subscribe to a feed by toggling the "Subscribe" switch, it will appear in the "Feeds" tab in the left navigation menu and in the "Custom" section of the <a href="/#browse">Browse page.</a></p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col s8">
+          <p class="promo-caption">Step 2: Categorization</p>
+          <p class="light">Our algorithm with automatically analyze each article in the feeds you subscribe to and assign the categories that best match. Check out the "Categories" tab in the navigation menu to the left and click on a Category to start reading. No work required!</p>
+        </div>
+        <div class="col s4">
+          <div class="center promo">
+            <i class="mdi-image-grain promo-icon"></i>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col s4">
+          <div class="center promo">
+            <i class="mdi-content-sort promo-icon"></i>
+          </div>
+        </div>
+        <div class="col s8">
+            <p class="promo-caption">Step 3: Reading!</p>
+            <p class="light">Customize your news reading experience to make Sourcemash work for you. Click on the "Categories" or "Feeds" tab in the navigation menu to the left, and select a category or feed to start reading!</p>
```

Comment:
Instead of reading articles by feed, you can now start reading by category! So...what are you waiting for? It's time to read!